infra/base-images/base-builder/precompile_centipede

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -eu
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+echo -n "Precompiling centipede"
+
+# Add Clang 15.
+wget https://commondatastorage.googleapis.com/chromium-browser-clang/Linux_x64/clang-llvmorg-15-init-1995-g5bec1ea7-1.tgz --output-document=/clang-llvmorg-15-init-1995-g5bec1ea7-1.tgz
```

Comment:
Oh no. Does centipede require a new compiler? This is really bad and we shouldn't be forcing two different compilers on our users. @oliverchang What do you think?

Reply:
I think we simply need to roll base-clang to clang 15. If AFL can't have it then AFL should be on its own compiler. Centipede will eventually be the primary fuzzer of oss-fuzz, so long term this makes sense.

Reply:
I ask the above question because you seem to use this compiler for `CC` `CXX` and `CCC` in compile_centipede.

Reply:
> roll base-clang to clang 15

That might not be necessary: Pasting [a discussion with Kostya](https://github.com/google/centipede/issues/80#issuecomment-1191020022) here.
I switched to `clang-15` after seeing a compilation bug related to dataflow tracing, but clang-14 (the default version in `OSS-FUZZ`) should already support it.
I will double-check it tomorrow.

Reply:
A quick update:
[Centipede's installation script](https://github.com/google/centipede/blob/79896174da9287874d8aa3ee969a0e8040dc80df/install_dependencies_debian.sh#L47) automatically downloads `Clang-14`, which is the oldest version of `Clang` that supports dataflow tracing.